Mapsui.Geometries/ArrowLine.cs

```diff
@@ -0,0 +1,196 @@
+﻿using Mapsui.Geometries.Utilities;
+using System;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Mapsui.Geometries
+{
+    /// <summary>
+    ///     An ArrowLine is an oriented line with an arrow as direction indicator 
+    /// </summary>
+    public class ArrowLine : MultiLineString
+    {
+
+        /// <summary>
+        ///     Initializes an instance of an ArrowLine
+        /// </summary>
+        public ArrowLine() : base() { }
+
+        /// <summary>
+        ///     Initializes an instance of an ArrowLine from two points, and specifications for the arrow (angle and length)
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="target"></param>
+        /// <param name="degreeAngle"></param>
+        /// <param name="arrowLength"></param>
+        public ArrowLine(Point source, Point target, double degreeAngle = 45, double arrowLength = 10)
+        {
+            double radiansAngle = Algorithms.DegreesToRadians(degreeAngle);
+            Angle = degreeAngle;
+            LineStrings = new Collection<LineString>();
+
+            LineString line = new LineString(new Point[] { source, target });
+            LineStrings.Add(line);
+
+            LineString firstArrowSide = new LineString(new Point[] { ArrowHead, ArrowExtremities(radiansAngle, arrowLength)[0] });
+            LineString secondArrowSide = new LineString(new Point[] { ArrowHead, ArrowExtremities(radiansAngle, arrowLength)[1] });
+            LineStrings.Add(firstArrowSide);
+            LineStrings.Add(secondArrowSide);
+
+        }
+
+        /// <summary>
+        ///     Initializes an instance of an ArrowLine from coordinates of two points
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="target"></param>   
+        public ArrowLine(double[] source, double[] target) : this(new Point(source), new Point(target)) { }
+
+        /// <summary>
+        ///    the arrow's Angle to the line (in radians)
+        /// </summary>
+        public double RadiansAngle
+        {
+            get
+            {
+                return Algorithms.DegreesToRadians(Angle);
+            }
```

Comment:
Since Angle has both "get" and "set", shouldn't we also have a RadiansAngle "set" property in order to allow for setting Angle in both Radians and Degrees?

Reply:
Now that's I've seen the whole code, I'm not convinced Angle and RadiansAngle should have "set" accessors. Other properties are get-only, why aren't these too?